universal-application-tool-0.0.1/app/services/question/InvalidPathException.java

```diff
@@ -0,0 +1,7 @@
+package services.question;
+
+public class InvalidPathException extends Exception {
+  public InvalidPathException(String pathName) {
+    super("Path not found: " + pathName);
```

Comment:
Thought (non-blocking): when we do versioning we'll likely want to include the version here as well